src/frontend.cs

```diff
@@ -1137,7 +1137,7 @@ public override object VisitVarPostOpAssign(bhlParser.VarPostOpAssignContext ctx
       FireError(Location(ctx.NAME()) + " : symbol not resolved");
 
     if(SymbolTable.IsRelopCompatibleType(vlhs.type.type) == false)
-      throw new UserError(Location(ctx.NAME()) + " : PostOpAssign not supported for string");
+      throw new UserError(Location(ctx.NAME()) + " : Invalid operand types");
```

Comment:
2 notes: 
 1) error messages start with lowercase 
 2) let's make it even more specific and clear

So it rather should be: incompatible variable type